tests/settings/ConfigurationGatheringTest.py

```diff
@@ -14,7 +14,8 @@
 from coalib.output.printers.LogPrinter import LogPrinter
 from coala_utils.string_processing import escape
 from coalib.settings.ConfigurationGathering import (
-    find_user_config, gather_configuration, load_configuration)
+    find_user_config, gather_configuration, load_configuration,
```

Comment:
One per line, in alpha order, with trailing comma and trailing round bracket on a separate line.